### ISSUE TITLE
[Dupe Bug & Suggested Solution] Dupe Bug for Dropping Items From Blocks

### ISSUE THREAD

**Issue by LoneDev6** [PluginBugs/Issues-ItemsAdder#85]
> [<img alt="PiggiesGoSqueal" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/PiggiesGoSqueal) **Authored by [PiggiesGoSqueal](https://github.com/PiggiesGoSqueal)**
_<time datetime="2019-11-15T11:40:17Z" title="Friday, November 15th 2019, 12:40:17 pm +01:00">Nov 15, 2019</time>_
_Closed <time datetime="2020-04-11T23:06:32Z" title="Sunday, April 12th 2020, 1:06:32 am +02:00">Apr 12, 2020</time>_
---

The default setup has a chance for crystals to drop when breaking nether quartz ore. This allows players to farm it using silk touch to silk the nether ore, then mine it to drop crystals, and repeat. The wiki does offer a feature for this:
drop_on_silktouch: false
However, it appears to only support the block settings and not the item settings. In other words, it'll prevent users from silk touching the nether quartz ore block which makes it impossible to obtain nether quartz ore as a block which we don't want. We want crystals to not drop when the block is silk touched.

In other words, could you add support for:
"drop_on_silktouch: false"
for the mineral section like so:
https://hastebin.com/bowecagusu.bash
